cli/config/configfile/file.go

```diff
@@ -186,6 +195,27 @@ func (configFile *ConfigFile) Save() error {
 	return configFile.SaveToWriter(f)
 }
 
+// ParseTrustPinning parses Notary trust pinning structure from configuration, and prepends CA certificate files with
+// the absolute path of the configuration directory
+func (configFile *ConfigFile) ParseTrustPinning() trustpinning.TrustPinConfig {
```

Comment:
Shouldn't this method replace in place the TrustPinning config instead of returning a new one? WDYT @vdemeester ?